Assets/LeapMotion/Core/Scripts/XR/XRHeightOffset.cs

```diff
@@ -101,11 +103,21 @@ public class XRHeightOffset : MonoBehaviour {
           autoRecenterOnUserPresence = false;
         }
       }
+#else
+      var trackingSpaceType = UnityEngine.VR.VRDevice.GetTrackingSpaceType();
```

Comment:
As for LeapXRServiceProvider, we should avoid having ifdefs and use XRSupportUtil so our cross-version support is cleaner.